ext/simplexml/tests/bug81325.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+BUg #81325 (segfault in zif_simplexml_import_dom)
+--SKIPIF--
+<?php
+if (!extension_loaded('simplexml')) die('skip simplexml not available');
+if (!extension_loaded('dom')) die('skip simplexml not available');
```

Comment:
skip message is incorrect

can this use EXTENSIONS section ?

Reply:
EXTENSIONS is used as of 8.1 only.

Reply:
> skip message is incorrect

@krakjoe, typo fixed.

> can this use EXTENSIONS section ?

This is only 8.1, right ?

Reply:
ah right, missed target ... so in master it should use EXTENSIONS, and in 8.0 the skip message should change for dom ....